### PR TITLE
Remove Proper from template

### PR DESCRIPTION
**Pull request by owaaquino**
Remove Proper from template to fix JS error on community.

Card: https://app.clickup.com/t/c690dx

### CODE REVIEW

File: assets/javascripts/discourse/templates/connectors/discovery-list-container-top/proper_ad.hbs [hunk 1]
```diff
@@ -1,5 +0,0 @@
-<div class="proper-ad-unit">
-  <div id="proper-ad-sitepoint_community_home">
-    <script> propertag && propertag.cmd.push(function () { proper_display('sitepoint_community_home'); }); </script>
-  </div>
-</div>
```
Comment (auraesteban) [inline assets/javascripts/discourse/templates/connectors/discovery-list-container-top/proper_ad.hbs:5]: Please add a new line.

Comment (mrlagmer) [reply assets/javascripts/discourse/templates/connectors/discovery-list-container-top/proper_ad.hbs:5]: The entire file was deleted nothing to add.

**Review by mrlagmer — CHANGES_REQUESTED** [review]
As you have deleted that file you would need to remove the folder: discovery-list-container-top/

File: assets/javascripts/discourse/templates/connectors/discovery-list-container-top/proper_ad.hbs (hunk 1 above)
Comment (auraesteban) [reply assets/javascripts/discourse/templates/connectors/discovery-list-container-top/proper_ad.hbs:5]: Ah, yeah. Sorry!

**Comment by owaaquino** [comment]
> As you have deleted that file you would need to remove the folder: discovery-list-container-top/

I've remove the folder already, not sure why it's not showing in the changes. Will look into it. Thanks.

**Review by mrlagmer — APPROVED** [review]

**Review by auraesteban — APPROVED** [review]

**Pull request merged**